config/rbac/role.yaml

```diff
@@ -5,6 +5,26 @@ metadata:
   creationTimestamp: null
   name: manager-role
 rules:
+- apiGroups:
```

Comment:
duplicated. This `configmaps` resource has already been assigned to permissions.

Reply:
done. Thank you